diode-proto/diode/v1/reconciler.proto

```diff
@@ -111,10 +111,38 @@ message RetrieveIngestionLogsResponse {
   string next_page_token = 3; // Token for the next page of results, if any
 }
 
+// The request to take action on an ingestion log
+message ActionIngestionLogRequest {
+  message DiffAction {}
+  message ApplyAction {}
+  message RejectAction {}
```

Comment:
@mfiedorowicz you mentioned using a `oneof` structure to represent this -- I imagine so that we can reserve space for varying arguments -- arguably the branch_id could belong to DiffAction and ApplyAction only if we prefer?

Reply:
That's correct, so we can either keep ingestion log ID and branch ID in the root of the message and replace these empty action type messages with i.e. `enum ActionType` that would define supported action types, especially if the branch ID is optional and making a new message just to hold branch ID might be excessive.